lib/UnoCore/Targets/Android/Android.uxl

```diff
@@ -226,6 +226,9 @@
     <ProcessFile Name="app/build.gradle" />
     <ProcessFile Name="app/src/main/AndroidManifest.xml" />
     <ProcessFile Name="app/src/main/CMakeLists.txt" />
+    <CopyFile Condition="Android" Name="@(Project.Android.Splash.SplashLogo:Path || '@//Assets/DefaultIcon.png')" TargetName="app/src/main/res/drawable/splash_logo.png" />
```

Comment:
Should `splash_logo.png` be in specific dimensions? If so, maybe we can use `<ImageFile>` instead, and pass `TargetWidth` and `TargetHeight`.

Reply:
I think its upto the designer to make their logo larger or smaller no?

Reply:
Would you prefer project properties for the width and height?

Reply:
Hmm, instead of project properties, it's probably better to use the logo as-is, yeah. Maybe the log will be automatically centered/resized by Android if dimensions aren't right?

Reply:
Yeah, we could put more constraints on the image here: https://github.com/fuse-open/uno/blob/4523619fc786cdd19c2818fd3075e215fdf61cb9/lib/UnoCore/Targets/Android/app/src/main/res/drawable/splash_background.xml

Just don't how if there is such a thing.

Reply:
I see, yeah, it's probably fine as-is. I think we can add more constraints and expose those as project properties, if we later need more control over that stuff.

Reply:
Alrighty